firstvoices/backend/serializers/job_serializers.py

```diff
@@ -84,38 +74,25 @@ def get_latest_recalculation_result(obj):
         return ordered_result
 
     class Meta:
-        model = CustomOrderRecalculationResult
-        fields = [
-            "site",
-            "current_task_status",
-            "latest_recalculation_date",
-            "latest_recalculation_result",
-        ]
+        model = DictionaryCleanupJob
+        fields = BaseJobSerializer.Meta.fields + ("is_preview", "cleanup_result")
 
 
-class CustomOrderRecalculationPreviewResultSerializer(
-    CustomOrderRecalculationResultSerializer
-):
-    current_preview_task_status = serializers.SerializerMethodField()
-    latest_recalculation_preview_result = serializers.JSONField(
-        source="latest_recalculation_result"
+class DictionaryCleanupPreviewJobSerializer(DictionaryCleanupJobSerializer):
+    url = SiteHyperlinkedIdentityField(
+        read_only=True, view_name="api:dictionary-cleanup-preview-detail"
     )
-    latest_recalculation_preview_date = serializers.DateTimeField(
-        source="latest_recalculation_date"
+    cleanup_preview_result = serializers.JSONField(
+        read_only=True,
+        source="cleanup_result",
     )
```

Comment:
Since this is a dedicated serializer for preview jobs, could we set is_preview to True in here?